### DISCUSSION TITLE
Gotek s760 issue!

### DISCUSSION THREAD

**Discussion by shindoja** [keirf/flashfloppy#715] [Q&A]
Hi ! Just got my Gotek SFRKC30.AT2, unmodified, plugged it in the s760, set jumpers to JB and SO. Its say please change disk after showed for .5sec ( Not s760 disk ).

I did format the usb to Master Boot Record, And add the 2.24.OUT file on Roland Website in the root directory also. 

Im far from being a code guy hehe   

Do i have to put another firmware in the Gotek or do i have to add the FF.CFG file into the usb also ? Dont wanna mess the firmware if i dont need to ! 

What im I missing to make it work ? 

Thanks a lot !

**Comment by keirf** [comment]
It sounds like you are using factory firmware and somewhat following this page: http://llamamusic.com/s50s550/S-760_USBFDE.html

It might work if you rename the file(s) as .IMG rather than .OUT?

Better still, reprogram your Gotek with FlashFloppy. You need a USB-A to USB-A cable, a Windows PC running approx Win8 or later, and a piece of wire/paperclip/etc to connect two PCB holes during programming. See https://github.com/keirf/FlashFloppy/wiki/Firmware-Programming

With FlashFloppy programmed, jumper S0 ONLY. Do not jumper JB. You should now be able to use files named .OUT just fine.

**Reply by shindoja** [reply]
Ok Thank you very much, ill try that ! ( And sorry for the issue on the other page not familiar with this site )

**Reply by shindoja** [reply]
Oh it work ! Thanks so much ! So if i want to make the rotary and Oled Upgrade will I just need to put the FF.CFG file with the right options in the usb boot ?

**Reply by keirf** [reply]
Yes. Although in many cases the oled and rotary will work with no configuration at all.

**Reply by shindoja** [reply]
Thanks again !
